### ISSUE TITLE
Already Joined User Still Sees "You're Invited" on Project

### ISSUE THREAD

**Issue by acshields** [appirio-tech/connect-app#4303]
**Bug Conditions to Reproduce**
I invited one of my personal email test accounts (username: "arieltest") to [this project](https://connect.topcoder.com/projects/26833/).
I received the email invitation, clicked "Join Project, and went into the project.
Later, I signed back into Projects App (Connect) and when viewing the 'all projects dashboard' I saw that the "You're Invited" (Join Y/N) workflow indicated on the project. 

Update **Required**
If the user accepts the invitation and joins the project from the email, the "You're Invited" workflow should no longer display on the project in 'all projects' dashboard.

**Comment by maxceem** [comment]
If after reloading the page this bug still persists, I guess this is an ES issue, and we have to try reindexing the project.
FYI @vikasrohit

**Comment by vikasrohit** [comment]
Yes, totally agree with @maxceem. @acshields are you sure you did a page refresh or went back to the project listing page using in app back button? If you are still seein `Invited` workflow for that same project on the listing page, let us know.

**Comment by acshields** [comment]
@maxceem @vikasrohit - Here's what happened:
-I received the email invitation, accepted it, and joined the project
-The next day, I opened a new browser tab logged in as "arieltest" and looked at my all projects dashboard

That's when I noticed the "invite" workflow was still displaying. Unfortunately, like a robot I just clicked "join" without thinking and didn't get a screenshot. I can try to reproduce it with a different project if that's helpful.

**Comment by maxceem** [comment]
@vikasrohit this looks like ES issue:
- to fix it, we can reindex this project
- to debug it, we might check logs related to this project or related to the invitation or member id in the ES Processor. This way we could try find out which data inside the project caused the issue with reindexing
- the 3rd way to debug is trying to run our script we've created to compare data in ES and DB and see it's output. It also has to contain data regarding this challenge if there are ES/DB inconsistencies

**Comment by vikasrohit** [comment]
Thanks @acshields I guess, if you were able to join the project second time and now you are not seeing the Join button, it must have been temporary glitch (ES inconsistency) which got fixed when you performed the operation second time and we can close this issue. However, if you are still able to reproduce it (just try to unjoin and then join again), it might be an bug.

**Comment by acshields** [comment]
@vikasrohit @maxceem - I was able to replicate this in a different project. Here are the steps I followed:

Invited "arieltest" (ariel.c.shields@gmail.com) to [this project.](https://connect.topcoder.com/projects/26349)
<img width="706" alt="Screen Shot 2021-02-22 at 12 28 03 PM" src="https://user-images.githubusercontent.com/41118149/108766399-f9bcfd80-7509-11eb-95c3-c4999d54d82b.png">

Accepted invitation in email, and was brought into the project.
<img width="1280" alt="Screen Shot 2021-02-22 at 12 28 14 PM" src="https://user-images.githubusercontent.com/41118149/108766454-06d9ec80-750a-11eb-8b86-c302b03c993a.png">

Logged out of Connect, logged back in as "arieltest" (ariel.c.shields@gmail.com). When viewing the all projects dashboard, I I see this:
<img width="453" alt="Screen Shot 2021-02-22 at 12 29 04 PM" src="https://user-images.githubusercontent.com/41118149/108766700-56201d00-750a-11eb-888d-c6d1b4677511.png">
<img width="455" alt="Screen Shot 2021-02-22 at 12 30 56 PM" src="https://user-images.githubusercontent.com/41118149/108766714-59b3a400-750a-11eb-9ffe-7ac865054af3.png">

**Comment by maxceem** [comment]
@acshields this is great that you could reproduce it again. Is it possible to invite me to this project by `maxceem`? So I would check it out, as I don't have access at the moment.

**Comment by acshields** [comment]
@maxceem - just invited you!

**Comment by maxceem** [comment]
@acshields thanks for inviting. Could you please check one more time if you can still see the invitation "Join/Decline" buttons for this project? Please, don't change anything, if you can still see the buttons, we would like to keep the project as it is to catch this error.

**Comment by acshields** [comment]
@maxceem - I just logged back in with "arieltest" and do not see the "you're invited" on the project anymore. I didn't change anything this time, but I do not recall if I had done anything the last time I logged in (when I took those screenshots above).
